### ISSUE TITLE
[ANNOUNCEMENT] Week 2! Reviewing assessments and introducing impact proposals

### ISSUE THREAD

**Issue by joenash** [campus-experts/spring-2017#8]
## The TLDR this week:

- Submit a [Pull Request](https://help.github.com/articles/creating-a-pull-request/) with your community assessment. 
- Check out and comment on other people's [community assessments](https://github.com/campus-experts/spring-2017/pulls?q=is%3Apr+is%3Aopen+label%3Acommunity-assessment).
- Begin creating your [Community Impact Proposal](https://github.com/campus-experts/spring-2017/blob/master/docs/community-impact-proposal.md)

## Welcome to Week 2.

It's so exciting to have you folks join us! Your energy in the first calls and effort you put into your bios and profile pictures is amazing. I'm really excited to start reading your community assessments!

A couple of things I want to reiterate here:

- Your community assessments do not need to be perfect before you complete the pull request. These are living documents, and they'll change in response to feedback and developing ideas! Faster you get the PR in, faster everyone can chip in :sparkles: 
- Campus Experts is not a solo-mission. Ask for help, ask for feedback, from us at GitHub and your fellow experts. If you see another Expert is working on similar problems to you - collaborate! 
- Timelines and deadlines. You should aim to have your activity for the week done before you intend to start the next activity. This is because activities will often build on one another, and also to help prevent you getting snowed under!
- **It's okay to be busy**! You'll have exams, deadlines, hackathons, real life. If something comes up and you can't complete an exercise or attend a call, don't sweat it, but do let us know if you can!

**Comment by joenash** [comment]
There was a weird duplicate issue bug here
